tools/karpenter-convert/pkg/karpenter-convert/karpenter-convert.go

```diff
@@ -0,0 +1,163 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package convert
+
+import (
+	"fmt"
+
+	"github.com/aws/karpenter/pkg/apis"
+	"github.com/aws/karpenter/pkg/apis/v1alpha1"
+	"github.com/aws/karpenter/pkg/apis/v1beta1"
+	nodeclassutil "github.com/aws/karpenter/pkg/utils/nodeclass"
+	"github.com/samber/lo"
+	"github.com/spf13/cobra"
+
+	"github.com/aws/karpenter-core/pkg/apis/v1alpha5"
+	corev1beta1 "github.com/aws/karpenter-core/pkg/apis/v1beta1"
+	nodepoolutil "github.com/aws/karpenter-core/pkg/utils/nodepool"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	"k8s.io/cli-runtime/pkg/genericiooptions"
+	"k8s.io/cli-runtime/pkg/resource"
+	"k8s.io/kubernetes/pkg/printers"
```

Comment:
I tried running this tool locally and it didn't succeed for me because we are relying on a package through a means that isn't the "supported" means to import upstream k8s packages into repos. We should be using the `k8s.io/cli-runtime/printers` package here rather than the upstream repo.

Importing the upstream repo breaks down in a bunch of places since a bunch of packages have their own modules and the upstream top-level project does a bunch of local replaces

Reply:
Done